languages.toml

```diff
@@ -703,6 +703,7 @@ scope = "source.jsx"
 injection-regex = "jsx"
 language-id = "javascriptreact"
 file-types = ["jsx"]
+roots = [ "package.json" ]
```

Comment:
Should this be added for JavaScript as well?

Reply:
Ohh yeah, that was definitely my intention but somehow that wasn't inlcuded in the commit. Should be fixed now!